c7n/commands.py

```diff
@@ -24,8 +24,16 @@
 from c7n.credentials import SessionFactory
 from c7n.policy import Policy, load as policy_load
 from c7n.reports import report as do_report
+from c7n.policymetrics import policy_metrics as do_policy_metrics
 from c7n.utils import Bag
-from c7n import mu, schema, version
+from c7n.exceptions import ArgumentError
+from c7n.manager import resources
+from c7n.resources import load_resources
+from c7n.schema import json_dump as schema_json_dump
```

Comment:
ew.. this is a lot import renaming.. just import the module and use the functions namespaced by it. 

from c7n import schema

schema.validate
shcema.json_dump